samples/net/azure_iot_hub/boards/native_sim.conf

```diff
@@ -0,0 +1,72 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# General
+CONFIG_LOG_BUFFER_SIZE=2048
+CONFIG_HW_ID_LIBRARY=y
+CONFIG_ASSERT=y
+CONFIG_FPU=y
```

Comment:
Why do we override these in this project specifically, but not the other two?

Reply:
removed